### DISCUSSION TITLE
Помогает ли это приложение справиться с этими блокировками?

### DISCUSSION THREAD

**Discussion by ghost** [dovecoteescapee/ByeDPIAndroid#72] [General]
![IMG_20240810_222121_849](https://github.com/user-attachments/assets/0d2baff5-a82a-4193-b7fd-b2d49f0f9f4b)
![IMG_20240810_222125_752](https://github.com/user-attachments/assets/7dc31689-26e1-4654-8035-5f52e726f638)

**Comment by dovecoteescapee** [comment]
Ваша нейросеть намешала разного. Из упомянутого техническими методами реализации блокировок являются:
1. **Блокировка IP**. Не помогает.
2. **Блокировка уровня DNS**. Для её обход достаточно не использовать DNS провайдера, что и делает приложение в режиме VPN (по умолчанию 1.1.1.1). Некоторые провайдеры перехватывают DNS запросы, в таком случае вам следует использовать DoT, в настройках Android он называется Private DNS. [Инструкция по настройке Private DNS](https://developers.cloudflare.com/1.1.1.1/setup/android/#android-11-or-later).
3. **Блокировка на основе анализа пакетов (DPI)**. Блокировки по домену реализуются этим методом. Именно для обхода данного типа блокировок разрабатывается ByeDPI.

**Reply by ghost** [reply]
> Ваша нейросеть намешала разного. Из упомянутого техническими методами реализации блокировок являются:
> 
> 1. **Блокировка IP**. Не помогает.
> 2. **Блокировка уровня DNS**. Для её обход достаточно не использовать DNS провайдера, что и делает приложение в режиме VPN (по умолчанию 1.1.1.1). Некоторые провайдеры перехватывают DNS запросы, в таком случае вам следует использовать DoT, в настройках Android он называется Private DNS. [Инструкция по настройке Private DNS](https://developers.cloudflare.com/1.1.1.1/setup/android/#android-11-or-later).
> 3. **Блокировка на основе анализа пакетов (DPI)**. Блокировки по домену реализуются этим методом. Именно для обхода данного типа блокировок разрабатывается ByeDPI.

Связывается ли государство с разработчиками по поводу блокировки IP и DNS?
Требует ли этого от них государство?
Или технически

**Reply by icynstant** [reply]
> Связывается ли государство с разработчиками по поводу блокировки IP и DNS? Требует ли этого от них государство? Или технически

Государство предъявляет требования провайдерам

**Reply by dovecoteescapee** [reply]
> Связывается ли государство с разработчиками по поводу блокировки IP и DNS?
> Требует ли этого от них государство?
> Или технически

С разработчиками чего? Роскомнадзор обращается к владельцам сайтов с требованием удалить какую-то информацию, если она попала в [реестр запрещённых ресурсов](https://reestr.rublacklist.net/ru/).
Сейчас блокировки в России осуществляются на двух уровнях:
1. **Блокировки провайдерами**. Провайдеры обязаны блокировать все ресурсы, перечисленные в соответствующих реестрах.
2. **Блокировки на ТСПУ**. ТСПУ (техническими средствами противодействия угрозам) это DPI, которым управляет сам Роскомнадзор. На них блокируются ресурсы из реестров, не из реестров, протоколы VPN и не только, замедляется траффик и т.д. В общем-то, Роскомнадзор распоряжается ими как пожелает.
